frontend/src/model/beta/settings/ModelAccess.tsx

```diff
@@ -51,19 +56,23 @@ export default function ModelAccess({ model }: ModelAccessProps) {
   const entities = useMemo(() => {
     if (!users) return []
 
-    const userGroup = users.find((usrGroup) => usrGroup.kind === 'user')
-    if (userGroup) {
-      return userGroup.entities
-    } else {
-      return []
+    const userList = users.find((usrGroup) => usrGroup.kind === 'user')
+    const groupList = users.find((usrGroup) => usrGroup.kind === 'group')
```

Comment:
There's not going to be a 'group' type, instead there will be dynamically defined group labels as defined by each deployment.  Forcing people to change these values here seems like a drag.